src/pages/RequisitionsPage.js

```diff
@@ -26,60 +25,28 @@ const DATA_TYPES_SYNCHRONISED = ['Requisition'];
 * @prop   {Realm.Object}        currentUser   User object representing the current user logged in.
 * @state  {Realm.Results}       requisitions  Results object containing all Requisition records.
 */
-export class RequisitionsPage extends GenericPage {
+export class RequisitionsPage extends React.Component {
   constructor(props) {
     super(props);
-    this.state.requisitions = props.database.objects('Requisition');
-    this.state.sortBy = 'entryDate';
-    this.state.isAscending = false;
-    this.state.columns = [
-      {
-        key: 'serialNumber',
-        width: 2,
-        title: tableStrings.requisition_number,
-        sortable: true,
-      },
-      {
-        key: 'entryDate',
-        width: 1,
-        title: tableStrings.entered_date,
-        sortable: true,
-      },
-      {
-        key: 'numberOfItems',
-        width: 1,
-        title: tableStrings.items,
-        sortable: true,
-        alignText: 'right',
-      },
-      {
-        key: 'status',
-        width: 1,
-        title: tableStrings.status,
-        sortable: true,
-      },
-      {
-        key: 'delete',
-        width: 1,
-        title: tableStrings.delete,
-        alignText: 'center',
-      },
-    ];
-    this.dataTypesSynchronised = DATA_TYPES_SYNCHRONISED;
-    this.getFilteredSortedData = this.getFilteredSortedData.bind(this);
-    this.onNewRequisition = this.onNewRequisition.bind(this);
-    this.onRowPress = this.onRowPress.bind(this);
-    this.renderCell = this.renderCell.bind(this);
-    this.navigateToRequisition = this.navigateToRequisition.bind(this);
+    this.state = {
+      selection: [],
+    };
+    this.requisitions = props.database.objects('Requisition');
+    this.dataFilters = {
+      searchTerm: '',
+      sortBy: 'entryDate',
```

Comment:
In mSupply invoices are actually ordered by serialNumber if I recall correctly; switching this to that would sort of fix #362 . That said, maybe don't bother and chase the solution described in the issue within a separate PR, seeing as the mSupply changes are already done.